db/migrations/003_track_unsigned_pr.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE unsigned_pr
+(
+    Id         UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    repo_owner varchar(250) NOT NULL,
+    repo_name  varchar(250) NOT NULL,
+    pr_number  int          NOT NULL,
+    login_name varchar(250) NOT NULL,
+    ClaVersion varchar(10),
+    CheckedAt  timestamp    NOT NULL,
+    UNIQUE (repo_name, pr_number, login_name, ClaVersion)
```

Comment:
You are mixing snake_case and CamelCase, do you really care about that? Just a question

Reply:
Oooh. Nice catch. No, I didn't mean to mix my camels and snakes. I'd rather keep snakes in the db, and camels elsewhere. Thanks!

Reply:
Done: e83fcc8